frontend/requests/request_processing.html

```diff
@@ -26,8 +26,18 @@ <h4 style="margin: 0;">Confirmar Vínculo</h4>
                   alt="{{ requestCtrl.children.name }}"></img>
               </div>
               <div layout="column" layout-align="center start">
-                <b>{{ requestCtrl.children.name || requestCtrl.children.sender_name}}</b>
-                <span>{{ requestCtrl.children.institutional_email }}</span>
+                <b hide-gt-xs> {{ requestCtrl.limitString( requestCtrl.children.name ||
```

Comment:
Instead of call limitString, it would be better create a function to get the children name, that receives max size of the string and returns the formatted name